src/tests/ftest/util/soak_test_base.py

```diff
@@ -294,6 +299,79 @@ def harasser_job_done(self, args):
         self.harasser_results[args["name"]] = args["status"]
         self.harasser_args[args["name"]] = args["vars"]
 
+    def schedule_jobs(self):
+        """Schedule jobs with internal scheduler."""
+        debug_logging(self.log, self.enable_debug_msg, "DBG: schedule_jobs ENTERED ")
+        job_queue = multiprocessing.Queue()
+        jobid_list = []
+        node_list = self.hostlist_clients
+        lib_path = os.getenv("LD_LIBRARY_PATH")
+        path = os.getenv("PATH")
+        v_env = os.getenv("VIRTUAL_ENV")
+        env = ";".join([f"export LD_LIBRARY_PATH={lib_path}",
+                        f"export PATH={path}",
+                        f"export VIRTUAL_ENV={v_env}"])
```

Comment:
I recommend using EnvironmentVariables here. It's just a dictionary with some helper methods
```
from command_utils_base import EnvironmentVariables

env = EnvironmentVariables()
env['FOO'] = 'bar'
env_str = env.to_export_str()  # export FOO=bar;